docs/section23/Rule23-13.md

```diff
@@ -0,0 +1,35 @@
+
+# Airside System - Rule 23-13 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-13  
+**Rule Description:** If proposed design includes humidistatic controls then the baseline shall use mechanical cooling for dehumidification and shall reheat to avoid overcooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.18 Dehumidification (Systems 3 through 8 and 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. P-RMR is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+
+**Applicability Checks:**  
+- For each hvac_p in the P_RMR: `for hvac_p in P_RMR...HeatingVentilatingAirConditioningSystem:`
+    - get the heating system: `heating_system_p = hvac_p.heating_system`
+    - check to see if the heating system has a humidification type other than NONE, if so, there are humidification controls and the rule is applicable: `if heating_system_p.humidification_type != NONE:`
+        - the rule is applicable: `APPLICABLE`
+    - get the cooling system: `cooling_system_p = hvac_p.cooling_system`
+    - check to see if the cooling system has a humidification type other than NONE, if so, there are humidification controls and the rule is applicable: `if cooling_system_p.humidification_type != NONE:`
+        - the rule is applicable: `APPLICABLE`
+- if we make it down here without triggering an applicable result, the rule is not applicable: `NOT_APPLICABLE`
```

Comment:
Please change outcome to undetermined.
Also, as far as I understand, the current implementation triggers "undetermined" outcome for all systems if at least one proposed HVAC system has humidity controls. Should we modify the logic so that it only applies to the baseline systems that serve zones that are served by a proposed system with humidity control?

Reply:
updated